### ISSUE TITLE
doc/words/examples/minchat.py has incorrect import path

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5595]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)| @ashfall reported|
|-|-|
|Trac ID|trac#5595|
|Type|defect|
|Created|2012-04-02 15:19:53Z|

basechat, baseaccount and ircsupport are being imported from twisted.im, which does not exist.
Need to correct module-path.

Attachments:

* [5595-path-fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/21e/21edfe05b1cadd499e2eecf55413f65fc08042d8/fce7dd8ff29ffec3065dd8f284e955d09c1478d7.patch) (696 bytes) - added by ashfall on 2012-04-02 15:37:10Z - Fixed the import line. Combined multiple import statements. Added instruction to run the script.
* [5595-topfile.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/21e/21edfe05b1cadd499e2eecf55413f65fc08042d8/55e439316fddba7b9de612895973f019b2420f30.patch) (296 bytes) - added by ashfall on 2012-04-02 15:38:00Z - Added topfile

<details><summary>Searchable metadata</summary>

```
trac-id__5595 5595
type__defect defect
reporter__ashfall ashfall
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1333379993000000 1333379993000000
changetime__1333392930000000 1333392930000000
version__None None
owner__thijs thijs
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)|@ashfall removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

Thanks for your patch. I will apply `5595-path-fix.patch` with some coding standard fixes (docstrings on 3 lines). The news file can also simply be an empty `.misc` file, only major additions to [doc](doc) need a `.doc` file.

While testing I also noticed this error, but I'll open a new ticket for that:

```
Traceback (most recent call last):
  File "/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/internet/tcp.py", line 203, in doRead
    rval = self.protocol.dataReceived(data)
  File "/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/words/protocols/irc.py", line 2413, in dataReceived
    basic.LineReceiver.dataReceived(self, data.replace('\r', ''))
  File "/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/protocols/basic.py", line 564, in dataReceived
    why = self.lineReceived(line)
  File "/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/words/protocols/irc.py", line 2421, in lineReceived
    self.handleCommand(command, prefix, params)
--- <exception caught here> ---
  File "/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/words/protocols/irc.py", line 2465, in handleCommand
    method(prefix, params)
  File "/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/words/protocols/irc.py", line 2052, in irc_RPL_ISUPPORT
    self.supported.parse(args)
exceptions.AttributeError: IRCProto instance has no attribute 'supported'
```

Same for the broken reference in the `UserWarning`:

```
$ python doc/words/examples/minchat.py 
/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/words/im/__init__.py:8: UserWarning: twisted.im will be undergoing a rewrite at some point in the future.
```

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34057]) Apply 5595-path-fix.patch: Correct module imports in the Words minchat example.

Author: ashfall
Reviewer: thijs
Fixes: [#5595](https://github.com/twisted/twisted/issues/5595)
